README.md

```diff
@@ -43,33 +43,35 @@ Mapping the dependencies between scenarios is done by annotating the steps with
 
-The second scenario has a dependency to the first scenario, due to the condition stated in the given-step of the scenario. How this works is by evaluating the expressions according to this schema:
+The second scenario has a dependency to the first scenario, due to the given-step: _Given you have a blank postcard_. This is expressed by the `:IN:` expression stating that you need a postcard, that does not have a wish on it yet. How this works, is by evaluating the expressions according to this schema:
 
 * given-steps evaluate only the `:IN:` expressions
 * when-steps evaluate both the `:IN:` and `:OUT:` expressions
 * then-steps evaluate only the `:OUT:` expressions
 
-If evaluation of any expression fails or is False, then the scenario is rejected. By properly annotating all steps to reflect their impact on the system or its environment, you can model the intended relations between scenarios. This forms the specification model. You can keep your models clean by deleting terms (e.g. `del postcard`) that are no longer relevant. The step implementations use keywords to connect to the system under test to verify the specified behaviour.
+If evaluation of any expression fails or is False, then the scenario is rejected. By properly annotating all steps to reflect their impact on the system or its environment, you can model the intended relations between scenarios. This forms the specification model. The step implementations use keywords to connect to the system under test. The keywords perform actions or verify the specified behaviour.
+
+You can keep your models clean by deleting domain terms that are no longer relevant (e.g. `del postcard`). If multiple expressions are needed you can separate them in the `*model info*` by using the pipe symbol (`|`) or starting the next expression on a new line. A single expression cannot be split over multiple lines.
 
 There are three typical kinds of steps
 
 * __Stative__  
-  Stative steps express a truth value. Like _you have a blank postcard_ For these steps the `:IN:` and `:OUT:` expressions have identical conditions and the step implementation consists purely of checks.
+  Stative steps express a truth value. Like, _you have a blank postcard_. For these steps the `:IN:` expression is a condition. The `:OUT:` part is either identical to the `:IN:` condition or a statement. Statements, like assigning a new property, are useful to express the result of a scenario. If the when-action is setting the property, then you use a condition in the `:OUT:` part. The step implementation of stative steps consists purely of checks.
 * __Action__  
   Action steps perform an action on the system that alters its state. These steps can have dependencies in their `:IN:` conditions that are needed to complete the action. Statements in the `:OUT:` expressions indicate what changes are expected by executing this action.
```

Comment:
an action on the system or by the system

Reply:
I think in this case 'on' is the correct one. We are looking at it from an action-step perspective and action-steps are typically performed by an actor on a system. It is only after you execute the next (stative) check that you know whether the action was actually executed by the system.